tools/README.md

```diff
@@ -137,7 +161,9 @@ rapids-4-spark-tools_2.12-<version>.jar \
 /path/to/eventlog1 /path/to/eventlog2 /directory/with/eventlogs
 ```
 
-### Options
+Note: We suggest you also save the output of the `spark-submit` to a log file for troubleshooting.
+
+### Options (How to run `--help`)
```

Comment:
This seems out of place? What is the how to run —help? Can we add a statement saying help must be before the event logs files

Reply:
1. For the `spark-submit` saving output section, initially I put after "how to" section. Now I moved to "output" section.
2. Basically the "options" section is the output of "--help".  So I just changed "How to run --help" to "--help output".
3. Also put a words saying "--help should be before the trailing event logs." in the "options" section.

How is now?